extensions/2.0/Khronos/KHR_interactivity/Specification.adoc

```diff
@@ -0,0 +1,2956 @@
+- Object Model property value at the time of the successful node activation
+- Information needed for cubic Bézier spline evaluation derived from the node's input values
+- Target property value
+- Implementation-specific pointer to the `done` output flow of the node that has added this entry
+
+This array is initially empty and its maximum size is implementation-specific.
+
+When the `in` input flow is activated:
+
+1. Evaluate all input values.
+2. If the pointer or the pointer template with all its substitutions applied cannot be resolved,
+.. activate the `err` output flow and skip the next steps.
+3. If the `duration` input value is NaN, infinite, negative, or not convertible into an implementation-specific time type used for the internal interpolation start time value,
+.. activate the `err` output flow and skip the next steps.
+4. If any component of the `p1` or `p2` input values is NaN or infinite or if any of the first components of these input values is negative or greater than 1,
+.. activate the `err` output flow and skip the next steps.
+5. If starting a new pointer interpolation exceeds any implementation-specific limit,
+.. activate the `err` output flow and skip the next steps.
+6. If the _pointer interpolation state dynamic array_ contains an entry with the same resolved JSON Pointer value,
+.. remove it from the array.
+7. Using the implicitly-defined end points stem:[P_0 (0, 0)] and stem:[P_3 (1, 1)] together with the control points stem:[P_1] and stem:[P_2] provided via the input values construct a cubic Bézier easing function for the stem:[[0, 1]] input range.
+8. Add a new entry to the _pointer interpolation state dynamic array_ filling it with the required information based on the evaluated input values.
+9. Activate the `out` output flow.
+
+On each asset animation update, for each entry in the _pointer interpolation state dynamic array_:
+
+1. Compute the current input progress position _t_ as the time passed since the interpolation start divided by the interpolation's duration.
+2. If _t_ is less than or equal to zero
+.. skip the next steps.
+3. If _t_ is NaN or greater than or equal to 1,
+.. set the Object Model property to the target value;
+.. remove the current entry from the _pointer interpolation state dynamic array_;
+.. activate the `done` output flow linked to the current entry
+.. skip the next steps.
+4. Using the cubic Bézier spline information, compute the output progress position _q_ based on the _t_ value. This step implies that latexmath:[t \in [0; 1\]].
+5. Set the linked Object Model property to the new value computed as a linear interpolation between the original and the target property values using the output progress position _q_ as the interpolation coefficient.
+
+If the Object Model property is a quaternion, spherical linear interpolation expression **SHOULD** be used.
+
+Intermediate output progress values **MAY** be less than zero or greater than one.
+
+==== Animation Control Nodes
```

Comment:
can it be explicitly called out if animation from interactivity graph takes precedence over built-in animations if they are using same pointer or its up to implementation

Reply:
closing this one as done, this will be in the text soon